lua/mason-registry/remark-cli/init.lua

```diff
@@ -0,0 +1,11 @@
+local Pkg = require "mason-core.package"
+local npm = require "mason-core.managers.npm"
+
+return Pkg.new {
+    name = "remark-cli",
+    desc = [[Command line interface to inspect and change markdown files with remark.]],
+    homepage = "https://github.com/remarkjs/remark/tree/main/packages/remark-cli",
```

Comment:
Websites are generally preferred when available

```suggestion
    homepage = "https://remark.js.org/",
```